.github/workflows/assign-issue-to-project.yml

```diff
@@ -3,9 +3,15 @@ name: Assign to Project
 on:
   issues:
     types: [opened]
+    branches:
+      - main
   pull_request:
     types: [opened]
+    branches:
+      - main
```

Comment:
I think we wanted to experiment with adding the PRs to our project board.

Reply:
damn. of course